### PR TITLE
docs(feat): Clouddriver git pull feature in 1.26

### PR DESCRIPTION
**Pull request by aimeeu**
git pull support for git repo artifacts https://spinnaker.io/community/releases/next-release-preview/#git-pull-support-for-gitrepo-artifacts

### CODE REVIEW

File: setup/artifacts/gitrepo.md [hunk 1]
```diff
@@ -1,13 +1,19 @@
...
 sidebar:
   nav: setup
 ---
 
 {% include toc %}
 
-This shows how to configure a Git Repo artifact account so Spinnaker can use an entire repository as a single artifact.
+## Overview
+
+This shows how to configure a Git repo artifact account so Spinnaker can use an entire repository as a single artifact.
```
Comment (317brian) [inline setup/artifacts/gitrepo.md]: missing `,` or `that` next to `so`

**Review by 317brian — CHANGES_REQUESTED** [review]
LGTM minus the one minor thing. Thanks for adding a little more to the overview too.

File: setup/artifacts/gitrepo.md (hunk 1 above)
Comment (aimeeu) [reply setup/artifacts/gitrepo.md]: or both

**Review by 317brian — APPROVED** [review]

**Review by german-muzquiz — APPROVED** [review]
LGTM

**Pull request merged**